### ISSUE TITLE
SingleFile from Selenium Webdriver to download pages when logged in

### ISSUE THREAD

**Issue by cacosandon** [gildas-lormeau/SingleFile#1051]
**Is your feature request related to a problem? Please describe.**
Hello. Really amazing project. I know this is not a problem, just wanted to know how to do by myself.

The chome extension can save a single html from any page, including pages where I am logged in. If I run SingleFile from the CLI, obviously, I can't be logged in so isn't useful. 

**Describe the solution you'd like**
Is it possible to pass to SingleFile anything else than just an URL?

Or maybe use Selenium for logging in with my own method and then run SingleFile, automated?

Basically, anything that can save HTMLs on the background.

**Comment by gildas-lormeau** [comment]
You can pass cookies via the option `--browser-cookies`
